cogs/todo.py

```diff
@@ -45,22 +61,23 @@ async def on_message(self, message):
             msg += f"{todo}\n"
         msg += "\nTODOに追加しますか？"
         msg = await message.channel.send(msg)
-        await msg.add_reaction("👍")
-        await msg.add_reaction("👎")
-        def check(reac, user):
+        await msg.add_reaction(REACTIONS[0])
+        await msg.add_reaction(REACTIONS[1])
+
+        def check(reaction, user):
             if not user == message.author:
                 return False
-            if not str(reac.emoji) in ["👍", "👎"]:
+            if not str(reaction.emoji) in REACTIONS:
                 return False
             return True
-        
-        reac, _ = await self.bot.wait_for("reaction_add", check=check)
-        if str(reac.emoji) == "👍":
-            if not self.bot.data.get(str(message.author.id)):
-                self.bot.data[str(message.author.id)] = []
+
+        reaction, _ = await self.bot.wait_for("reaction_add", check=check)
+        if str(reaction.emoji) == REACTIONS[0]:
+            if not self.bot.data["todo"].get(str(message.author.id)):

```

Comment:
```suggestion
            if str(message.author.id) not in self.bot.data["todo"]:
```
get を使ってると、ぱっと見た時何書いてるかわからなかった

Reply:
has_key の代用で key in dict が出来るので、そちらのほうが良いと思われる